config.json

```diff
@@ -1,3 +1,4 @@
 {
-    "prefix": "cm!"
+	"prefix": "cm!",
+	"adminRole": "Moderators"
```

Comment:
I introduced a Admin-Role to make some commands only accessible to users with a specific role. This can be adjusted